scripts/preconversion_assessment_script.py

```diff
@@ -80,6 +80,54 @@ def to_dict(self):
         }
 
 
+def _check_ini_file_modified():
+    rpm_va_output, ini_file_not_modified = run_subprocess(
+        ["/usr/bin/rpm", "-Va", "convert2rhel"]
+    )
+
+    # No modifications at all
+    if not ini_file_not_modified:
+        return False
+
+    lines = rpm_va_output.strip().split("\n")
+    for line in lines:
+        line = line.strip().split()
+        status = line[0].replace(".", "").replace("?", "")
+        path = line[-1]
+
+        default_ini_modified = path == "/etc/convert2rhel.ini"
+        md5_hash_mismatch = "5" in status
+
+        if default_ini_modified and md5_hash_mismatch:
+            return True
+    return False
+
+
+def check_convert2rhel_inhibitors_before_run():
+    """
+    Conditions that must be True in order to run convert2rhel command.
+    """
+    ini_report = (
+        "Custom or modified ini file would cause re-registration of system to Red Hat Insights "
+        "therefore conversion was inhibited. "
+        "Please restore the /etc/convert2rhel.ini to its original state by running "
+        "'yum reinstall convert2rhel' or uninstall convert2rhel by running "
+        "'yum remove convert2rhel' "
+        "(running conversion task again will install convert2rhel again)."
```

Comment:
Reinstalling convert2rhel won't help because it is defined as a config file in the convert2rhel spec file meaning that it doesn't get replaced during the reinstallation when the config filed has been edited. That's intentional - updating or reinstalling a package should not reset your config file to the default values.

Removing convert2rhel and installing it again will help. When uninstalling convert2rhel with the ini file edited I see:
```
Running transaction
  Erasing    : convert2rhel-1.5.0-2.20231121201833740803.main.12.g7decb7e.el7.noarch                                                                                                                                                      1/1 
warning: /etc/convert2rhel.ini saved as /etc/convert2rhel.ini.rpmsave
```

From Miriam I know that we should not use "Please" :)
And better to not talk about "conversion task" when this is in fact the pre-conversion analysis task.

```suggestion
        "Either remove the /etc/convert2rhel.ini file by running "
        "'rm -f /etc/convert2rhel.ini' or uninstall convert2rhel by running "
        "'yum remove convert2rhel' before running the Task again."
```

Reply:
Ah alright, makes sense, updated the message both in preconversion and conversion!

Reply:
Also updated the separate errors messages (custom vs default ini problem) to include correct paths in the message.

@bocekm should be ready now